app/views/users/me.json.jbuilder

```diff
@@ -0,0 +1,4 @@
+json.partial! "user", user: @current_user
+json.partial! 'tags/tag', tags: @current_user.tags
+json.extract! @current_user, :email, :active, :created_at, :updated_at,:member_since
+json.set! :token, @token
```

Comment:
yep! In `user.rb` you could alias `refresh_token` to `token` and in the template here, just add `:token` to the `json.extract! @current_user`, i.e.
```ruby
# models/user.rb
def refresh_token
  ....whatever method we had
end
alias :token :refresh_token

# app/views/users/me.json.jbuilder
...
json.extract! @current_user, :email, :active, :token, :created_at, :updated_at,:member_since

```